_build/src/dep/GUM_whow_quidditch.conll10

```diff
@@ -888,19 +888,19 @@
 6	snitches	_	NNS	NNS	_	4	dobj	_	_
 7	.	_	SENT	.	_	0	punct	_	_
 
-1	You	_	PP	PRP	_	3	nsubj	_	_
-2	can	_	MD	MD	_	3	aux	_	_
-3	buy	_	VV	VB	_	0	root	_	_
-4	authentic	_	JJ	JJ	_	5	mwe	_	_
-5	looking	_	VVG	VBG	_	6	amod	_	_
-6	brooms	_	NNS	NNS	_	3	dobj	_	_
-7	to	_	TO	TO	_	8	aux	_	_
-8	make	_	VV	VB	_	3	vmod	_	_
-9	the	_	DT	DT	_	10	det	_	_
-10	game	_	NN	NN	_	12	nsubj	_	_
-11	more	_	RBR	RBR	_	12	advmod	_	_
-12	realistic	_	JJ	JJ	_	8	xcomp	_	_
-13	.	_	SENT	.	_	0	punct	_	_
+1	You	_	PP	_	_	3	nsubj	_	_
+2	can	_	MD	_	_	3	aux	_	_
+3	buy	_	VV	_	_	0	root	_	_
+4	authentic	_	JJ	_	_	6	amod	_	_
```

Comment:
If this were a clause, you'd have acomp: looking authentic. But since we've been treating these as compounds, it should be 'nn', and 'looking' is the head